tools/bwa.wdl

```diff
@@ -63,21 +63,21 @@ task bwa_aln {
             n_cores=$(nproc)
         fi
         # -1 because samtools uses one more core than `--threads` specifies
-        let "samtools_cores = $n_cores - 1"
+        (( samtools_cores = n_cores - 1 ))
 
         mkdir bwa_db
-        tar -C bwa_db -xzf ~{bwa_db_tar_gz} --no-same-owner
+        tar -C bwa_db -xzf "~{bwa_db_tar_gz}" --no-same-owner
         PREFIX=$(basename bwa_db/*.ann ".ann")
 
-        bwa aln -t "$n_cores" bwa_db/"$PREFIX" ~{fastq} > sai
+        bwa aln -t "$n_cores" bwa_db/"$PREFIX" "~{fastq}" > sai
 
         bwa samse \
-            ~{if read_group != "" then "-r '" + read_group + "'" else ""} \
+            ~{if read_group != "" then "-r '~{read_group}'" else ""} \
```

Comment:
not for this PR, but should _our_ `bwa` task require a read group be provided? What does `bwa` do when one isn't provided?

Reply:
#242